presto-pinot/presto-pinot-lib/src/main/java/com/facebook/presto/pinot/PinotConnectorId.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.pinot;
+
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+public final class PinotConnectorId
```

Comment:
We don't need this class. `ConnectorId` will be taken cared by presto engine. We can remove all usage of this class in the connector.

Reply:
Okay. Thanks for that ! Perhaps it was only needed in the previous presto versions. I will remove it.